backend/index.ts

```diff
@@ -1,12 +1,82 @@
 const express = require('express');
+const mongoose = require('mongoose');
+const bodyParser = require('body-parser');
+
 const app = express();
+app.use(bodyParser.json());
+
+// Connect to MongoDB
+mongoose.connect(
+  'mongodb://localhost/mydatabase',
+  { useNewUrlParser: true }
+);
+
+const itemSchema = new mongoose.Schema({
```

Comment:
Put the schema into a seperate file. 
Create a schema for Blog. 
Replace itemSchema with blogSchema.

Reply:
What would the schema look like for a blog? 
I'm thinking title, date n time, author, text. anything else?